bin/stencil-init.spec.js

```diff
@@ -14,7 +14,7 @@ describe('stencil init', () => {
     let sandbox;
 
     lab.beforeEach(done => {
-        sandbox = Sinon.sandbox.create();
+        sandbox = Sinon.createSandbox();
```

Comment:
Doing this cleanup as part of this PR to fix some deprecation notices. Only affects tests.